app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,52 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+h1,h2,h3,h4 {
+	color: #24C73A;
+}
+
+body {
+	background-color: #fedf6b;
+}
+
+.container-fluid {
+	background-image: url('adventure.jpg');
+	margin: 66px -129px 7px 10px;
+	background-size: cover;
+}
+
+a {
+	color: #0180c0;
+}
+
+h1 {
+	border-bottom: 1px solid #24C73A;
+	text-align: center;
+	padding: 10px;
+}
+
+.warning {
+		background-color: #00ccd3;
+		text-align: center;
+		font-size: 22px;
+		color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs